contrib/openshift/manifests/manifests.yaml

```diff
@@ -81,6 +81,14 @@ objects:
                   mountPath: /etc/clair
                 - name: indexer-layer-storage
                   mountPath: /tmp
+          initContainers:
+            - name: init-wipe-vpc
+              image: ${{UBI_IMAGE}}:${{UBI_IMAGE_TAG}}
+              command: ['sh', '-c', "rm -rf /tmp/*"]
```

Comment:
I was between this and adding the layer prefix. Thoughts?

Reply:
I'd maybe mount it somewhere else for the delete if you're not sure about the glob. Seems fine to me, though.